hubspot.com.hubspot-cname.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "hubspot.com",
+  "providerName": "HubSpot",
+  "serviceId": "hubspot-cname",
+  "serviceName": "HubSpot Site",
+  "version": 1,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
+  "description": "Enables a sub-domain to work with HubSpot",
+  "variableDescription": "cnameValue - cname record pointsTo value, cNameHostValue - cname host value",
+  "hostRequired": true,
+  "records": [
+    {
+      "groupId": "cname",
+      "type": "CNAME",
+      "host": "%cNameHostValue%",
```

Comment:
The same consideration as above - this template could however use `@` as host will determine the subdomain in fact.